Sitemap/Voter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * A chain as a decorator for all tagged voter services.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Voter extends AbstractChain implements VoterInterface
```

Comment:
@dbu i can't really follow your ideas with the `VoterInterface`. The `attribute` can be like:
```
[ sitemap => 'sitemap1, sitemap2, ..']
```
The object is the document/content to vote for, but what should i use as a token? I have got no security context here, is there an empty token (new) that i should use?